File: ETicaretClient/src/app/contracts/supplierOrder/listSupplierOrder.ts
import { listSupplierOrderItem } from "../supplierOrderItem/listSupplierOrderItem";

export class listSupplierOrder {
  id: string; 
  orderTrackingNumber: string; 
  userName: string;
  address: string;
  description: string;
  isCompleted: boolean;
  supplierId: string; 
  orderId: string; 
  totalPrice: number; 
  createdTime: Date;
  supplierOrderItems: listSupplierOrderItem[];
}

File: ETicaretAPI/Core/ETicaretAPI.Application/DTOs/SupplierOrder/ListSupplierOrder.cs
using ETicaretAPI.Application.DTOs.SupplierOrderItem;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ETicaretAPI.Application.DTOs.SupplierOrder
{
    public class ListSupplierOrder
    {
        public Guid Id { get; set; }
        public Guid OrderId { get; set; }
        public Guid SupplierId { get; set; }
        public Guid OrderTrackingNumber { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
        public bool IsCompleted { get; set; }
        public long TotalPrice { get; set; }
        public List<ListSupplierOrderItem> ListSupplierOrderItems { get; set; }
    }
}
